gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ClusterController.groovy

```diff
@@ -35,22 +35,22 @@ class ClusterController {
   @Autowired
   LoadBalancerService loadBalancerService
 
-  @ApiOperation(value = "Retrieve a list of cluster names for an application, grouped by account")
+  @ApiOperation(value = "Retrieve a list of cluster names for an application, grouped by account", response = HashMap.class)
   @RequestMapping(method = RequestMethod.GET)
   Map getClusters(@PathVariable("application") String app,
                   @RequestHeader(value = "X-RateLimit-App", required = false) String sourceApp) {
     clusterService.getClusters(app, sourceApp)
   }
 
-  @ApiOperation(value = "Retrieve a list of clusters for an account")
+  @ApiOperation(value = "Retrieve a list of clusters for an account", response = HashMap.class, responseContainer = "List")
```

Comment:
Why responseContainer?

Reply:
This lets us preserve the fidelity of the response classes as well as the fact they are in a collection. If we just use a `List.class`, `swagger-codegen` generated code parses this as a list of the most generic type in the target language, e.g. `interface{}` etc.